replace_address_test.py

```diff
@@ -78,6 +78,8 @@ def _do_replace(self, same_address=False, jvm_option='replace_address',
                     extra_jvm_args=None):
         if replace_address is None:
             replace_address = self.replaced_node.address()
+        if data_center is None:
+            data_center = "dc1"
```

Comment:
Why "dc1" was not added deterministically everywhere? What do I miss?

Reply:
Sorry, I just saw this change after I pressed approve

Reply:
Bc there is only one multi-dc test. The others work ok as they fallback to the default 'dc1' hard coded in the default topology file. But on multi-dc we need to be explicit.